boards/arm/sam_e70_xplained/doc/sam_e70_xplained.rst

```diff
@@ -141,6 +141,17 @@ Flashing
    The command will also verify that the image was programmed correctly, reset
    the board and run the Zephyr application.
 
+   It is also possible to use an external debug adapter such as J-Link or ULINK
```

Comment:
Thanks for the review. The corrected text reads better indeed. I updated the paragraph as suggested adding 'on its internal search path' at the end of it. This is to make it clear that the interface file is part of OpenOCD installation and not Zephyr. Hope that's fine.